### PR TITLE
data migration: continuous data validation goes GA

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

DM continuous data validation goes GA.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/12091
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by D3Hunter — APPROVED** [review]

File: dm/dm-continuous-data-validation.md [hunk 1]
```diff
@@ -5,10 +5,6 @@ summary: Learn how to use continuous data validation and the working principles
 
 # Continuous Data Validation in DM
```
Comment (shichun-0415) [inline dm/dm-continuous-data-validation.md:6]: Should we remove this feature from [experimental feature list](https://docs.pingcap.com/tidb/dev/experimental-features#data-migration) as well?

Comment (hfxsd) [reply dm/dm-continuous-data-validation.md:6]: 是的，但是那篇文档不是单独编辑修改的吗？不需要在这个 pr 里改吧。

**Pull request merged**